components/tabs/tab-mixin.js

```diff
@@ -70,26 +80,12 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 		this.tabIndex = -1;
 	}
 
-	connectedCallback() {
-		super.connectedCallback();
-		this.#addEventHandlers();
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
 
-		if (!this.#resizeObserver) {
```

Comment:
While working on the docs I realized that we don't use the resize observer, since we went in favour of the consumer dispatching a change event via a shared helper method. I cleaned this up and moved the other listeners to a `firstUpdated`. Let me know if there's a reason these were in `connectedCallback` instead - it seemed to work as before.